src/pkg/services/m365/api/drive.go

```diff
@@ -13,41 +13,42 @@ import (
 )
 
 // ---------------------------------------------------------------------------
-// Drives
+// Folders
 // ---------------------------------------------------------------------------
 
-func GetUsersDrive(
-	ctx context.Context,
-	srv graph.Servicer,
-	user string,
-) (models.Driveable, error) {
-	d, err := srv.Client().
-		Users().
-		ByUserId(user).
-		Drive().
-		Get(ctx, nil)
-	if err != nil {
-		return nil, graph.Wrap(ctx, err, "getting user's drive")
-	}
+const itemByPathRawURLFmt = "https://graph.microsoft.com/v1.0/drives/%s/items/%s:/%s"
 
-	return d, nil
-}
+var ErrFolderNotFound = clues.New("folder not found")
```

Comment:
minor nit: We use the same `Err*` prefix for graph errors as well as clues errors.  Might be good to separate them into different error prefixes.

Reply:
`Err*` simply suggests the value holds an `error`.  It doesn't matter which package created the value, so long as its compliant with that interface.